### PR TITLE
Axle 50 terminate session

### PR DESCRIPTION
**Pull request by Waqas-Zamir**
First I have created a hub method `terminateSession(accountId)`
But custom authorization handler doesn't work with SignalR.

Then I have added DELETE `api/sessions?accountId` endpoint

I will remove one of them as decided by the team

### CODE REVIEW

File: src/Axle/Hubs/SessionHub.cs [hunk 1]
```diff
@@ -51,6 +56,25 @@ public override Task OnDisconnectedAsync(Exception exception)
...
+            if (string.IsNullOrWhiteSpace(userId))
+            {
+                return new TerminateSessionResponse
+                {
+                    Status = TerminateSessionStatus.BadRequest,
+                    ErrorMessage = "Couldn't find account owner user id"
+                };
+            }
+
+            var result = await this.sessionLifecycleService.TerminateSession(userId);
```
Comment (dafigerdjikova) [inline src/Axle/Hubs/SessionHub.cs]: This will delete redis session storage as well, correct?

File: src/Axle/Services/AccountsService.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    public class AccountsService : IAccountsService
+    {
+        private readonly IAccountsMgmtApi accountsMgmtApi;
+
+        public AccountsService(IAccountsMgmtApi accountsMgmtApi)
+        {
+            this.accountsMgmtApi = accountsMgmtApi;
+        }
+
+        public async Task<string> GetAccountOwnerUserId(string accountId)
```
Comment (dafigerdjikova) [inline src/Axle/Services/AccountsService.cs]: we can add todo that this should be removed once we have accounts session management implementation.

File: src/Axle/Hubs/SessionHub.cs [hunk 3]
```diff
@@ -51,6 +56,25 @@ public override Task OnDisconnectedAsync(Exception exception)
             this.connectionRepository.Remove(this.Context.ConnectionId);
             this.sessionLifecycleService.CloseConnection(this.Context.ConnectionId);
             return base.OnDisconnectedAsync(exception);
+        }
+
+        [AuthorizeUser(Permissions.CancelSession)]
```
Comment (dafigerdjikova) [inline src/Axle/Hubs/SessionHub.cs]: If this attribute is not working let's remove SignalR implementation and leave only the API call

File: src/Axle/Hubs/SessionHub.cs (hunk 1 above)
Comment (Waqas-Zamir) [reply src/Axle/Hubs/SessionHub.cs]: yes

File: src/Axle/Services/AccountsService.cs (hunk 2 above)
Comment (Waqas-Zamir) [reply src/Axle/Services/AccountsService.cs]: done

File: src/Axle/Hubs/SessionHub.cs (hunk 3 above)
Comment (Waqas-Zamir) [reply src/Axle/Hubs/SessionHub.cs]: done

**Pull request merged**